frontend/src/export/index.js

```diff
@@ -0,0 +1,22 @@
+import cockatrice from "./cockatrice";
+import mtgo from "./mtgo";
+import mtga from "./mtga";
+import text from "./text";
+
+const toJSON = (filename, deck) => JSON.stringify(deck, null, 2);
+const json = {
+  name: "JSON",
+  download: toJSON,
+  downloadExtension: ".json",
+  copy: toJSON
+};
+
+export default {
+  cockatrice,
+  mtga,
+  mtgo,
+  text,
+  json: process.env.NODE_ENV === "development" ? json : null
```

Comment:
What is the problem with always outputting the json (not only on development mode) ? If it's fine for you, I'd keep it as always outputting the json.

Reply:
This json is different to the old one - it contains all the info the app is tracking about each card in the deck (set Code, identifiers, cmc, card number,....)

I like it on for debugging, but it might be pretty confusing for non coders.
For a "raw" format i made "text" output, which is simple and very readable.

Up for experimenting and turning it always on if you want

Reply:
The json file should not be relevant for end users IMO, but rather distracting and confusing.

Reply:
I'm gonna leave it disabled in production then